app/views/content/life-as-a-teacher/explore-subjects/history.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Become a history teacher
+subject: history
+title_paragraph: |-
+  <p>As a history teacher, you'll bring the past to life by introducing pupils to key events and significant people. By encouraging the use of critical thinking, you’ll help pupils form their own opinions and uncover new perspectives.</p>
```

Comment:
```suggestion
  <p>As a history teacher, you'll bring the past to life by introducing pupils to key events and significant people from history. By encouraging critical thinking, you’ll help pupils form their own opinions and uncover new perspectives.</p>
```

Reply:
Just a suggestion, maybe 'significant people from the past', just think it needs to say what kind of significant people